etc/kayobe/kolla/inventory/overcloud-components.j2

```diff
@@ -0,0 +1,201 @@
+# This inventory section provides a mapping of the components to top level
```

Comment:
Perhaps you could outline why we're using a custom inventory here, or alternatively add a comment next to lines that are changed from the original.

Reply:
Will do. I put a diff in the commit message to check nothing unexpected happened.

Reply:
Oh, yes I see now. I should make more effort to look at the individual commits, especially when you've gone to the effort of separating them nicely.

Reply:
No worries - github seems good at hiding the commit messages.